src/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "nativescript-iqkeyboardmanager",
+  "version": "1.3.0",
+  "description": "NativeScript wrapper of the popular IQKeyboardManager iOS library",
+  "main": "iqkeyboardmanager",
+  "typings": "index.d.ts",
+  "nativescript": {
+    "platforms": {
+      "ios": "3.0.0",
+      "android": "3.0.0"
```

Comment:
Hey @EddyVerbruggen , this plugin does not have any Android implementation, so it shouldn't be included in the Android projects. i.e. can you please remove the android key from here. This way NativeScript CLI will detect that this plugin shouldn't be included in the android project and it will skip it.

Reply:
Hi, I added a new view tag and want it to render on Android as well. So I figured I had to add Android support to package.json, or otherwise the view would not be recognized. Does that make sense?

Reply:
Hey @EddyVerbruggen, what's the purpose of the view? Do we have some IQKeyboardManager specific logic for Android? IMO in case the plugin does not have something to add to the Android project, it shouldn't have android key in the package.json (and it shouldn't have android demo as well).

Reply:
@rosen-vladimirov The purpose of the view (and this PR) is to make IQKeyboardManager provide next/previous keys in the accessory bar in case the TextFields aren't siblings (in which case it automagically works.

So in this case the buttons appear:
```html
<StackLayout>
  <TextField hint="Email"></TextField>
  <TextField hint="Password"></TextField>
</StackLayout>
```

And in this case they would not:
```html
<StackLayout>
  <StackLayout>
    <TextField hint="Email"></TextField>
  </StackLayout>
  <StackLayout>
    <TextField hint="Password"></TextField>
  </StackLayout>
</StackLayout>
```
So to enable that case, his new `PreviousNextView` wrapper was introduced, that exposes a similar wrapper view the IQKeyboardManager library offers (example from the updated readme):

```html
<StackLayout>
  <PreviousNextView><!-- add this 'wrapper' to enable those previous / next buttons -->
    <StackLayout>
      <StackLayout>
        <TextField hint="Email"></TextField>
      </StackLayout>
      <StackLayout>
        <TextField hint="Password"></TextField>
      </StackLayout>
    </StackLayout>
  </PreviousNextView>
</Stacklayout>
```

So what would happen if the user needs this feature and also needs to support Android? He'd have to define the UI **twice**, unless (AFAIK) we also add Android support the way I did.

Btw, what's the reason of your concern?

Reply:
Thanks for the great explanation @EddyVerbruggen . Sorry, I should have explained why I'm asking. We have some internal tests that verify the nativescript-iqkeyboardmanager plugin will not be used in the Android app. After yesterday, the tests started failing. So we were wondering why. I do not have any concerns, we will fix the tests, I was just wondering the purpose of the view and it is clear now. Sorry for the disruption and thanks again for your explanations.

Reply:
Awesome, thanks for those details. Your questions make perfect sense in that context. Keep up the great work! 👍